src/dfa/LG_Actions2.cpp

```diff
@@ -40,8 +40,8 @@ static int*   L_tail;
 static char*  ebnfspace;
 static char*  ebnfspace_start;
 
-static int    c;
-static int    warning_off = 0;
+//static int    c;
```

Comment:
Code that is commented out should be removed.  If needed, it can be recovered.

Reply:
In fact it need a more thorough investigation before remove it because there is several commented out code that maybe is using it when uncommented.